src/testsuite/vec_int128_dummy.c

```diff
@@ -692,10 +692,17 @@ test_vec_revbq (vui128_t a)
 void
 test_vec_load_store (vui128_t *a, vui128_t *b)
   {
+#ifdef __clang__
+    vui32_t temp;
+
+    temp = vec_ld (0, (vui32_t *)a);
+    vec_st (temp, 0, (vui32_t *)b);
+#else
```

Comment:
so, what's this all about? Is there a bug open against clang, if clang generates bad code otherwise?